code/modules/unit_tests/computer_smoothing.dm

```diff
@@ -0,0 +1,15 @@
+/datum/unit_test/computer_smoothing
+
+//This could potentially be revised to do an overall check for icon states of a smoothing atom to see if any are missing
+/datum/unit_test/computer_smoothing/Run()
+	var/atom/spawn_loc = run_loc_floor_bottom_left
+	var/obj/machinery/computer/base = /obj/machinery/computer
+	for(var/comp_type in subtypesof(/obj/machinery/computer))
+		var/obj/machinery/computer/comp = comp_type
+		if(isnull(initial(comp.smoothing_flags)))
+			Fail("[comp_type] has smoothing_flags set to null instead of NONE!")
+			continue
+		//This will potentially pickup false positives if someone makes a subtype to a non-smoothing subtype, which smooths and uses its own icon state names
+		//This is why we should refactor this to a smoothing test of smoothing states later on.
```

Comment:
if we're adding an incomplete test it is much better to just finish it now. It is unlikely that this test specifically would regress, a complete test would be the only useful test.

Reply:
Alright, but it might take extra time to figure out how to test all atoms that smooth so I'll just remove the Unit Test here alltogether and make a different PR with the unit test- currently all the machines here are invisible ingame when they shouldn't be.